lib/en/managing-domains-ssl.adoc

```diff
@@ -127,10 +127,21 @@ If you are still getting by on the link:https://www.openshift.com/products/prici
 
 image::overview-platform-features-15.png[SSL Certificate]
 
+*Note* If your SSL certificate includes a bundled file with primary certificate and intermediate certificate concatenated to a single file (a .crt or .pem file), this is the file you will need to upload.  If your SSL provider does not provide a concatenated file, you will need to manually concatenate the files.  From the directory on your server where your certificate files are, run the following command to create the concatenated file:
+
+[source]
+----
+$ cat your_primary_cert.pem your_intermediate_cert.pem >> fullchain.pem
```

Comment:
Might be better to use `>` so that rerunning recreates the file instead of catenating duplicate certs onto the existing file.  I don't feel strongly about that though.